src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -650,6 +763,21 @@ private ProcessWaitState GetWaitState()
             return _waitStateHolder._state;
         }
 
+        private static readonly string[] s_validVerbs = new string[] { null, "", "open" };
+
+        private static bool IsValidVerb(string verb)
+        {
+            for (int i = 0; i < s_validVerbs.Length; i++)
+            {
+                if (s_validVerbs[i] == verb)
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
```

Comment:
If above suggestions are used, this can be removed.
https://github.com/dotnet/corefx/blob/a11365cdab5108cc2d435322db0595e04145b4cd/src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs#L766-L779

Reply:
I'd go somewhere in between:

```csharp
private static bool IsValidVerb(string verb) 
{
  return verb == ...;
}
```